pkg/controllers/csi/driver/volumes/app/publisher.go

```diff
@@ -139,15 +139,15 @@ func (publisher *AppVolumePublisher) fireVolumeUnpublishedMetric(volume metadata
 	}
 }
 
-func (publisher *AppVolumePublisher) buildLowerDir(bindCfg *csivolumes.BindConfig) string {
+func (publisher *AppVolumePublisher) buildLowerDir(bindCfg *csivolumes.BindConfig, dynakubeName string) string {
```

Comment:
I would rather include the dynakube name into the bindCfg instead, as we are in control of that struct and it already has a lot of dynakube specific attributes :)

Reply:
[Fixed](https://github.com/Dynatrace/dynatrace-operator/pull/2583/commits/2433ba5e9238f667b34fd699125d0bc4fc313be3)